File: app/src/main/java/hk/collaction/timber/ui/activity/SettingsActivity.java
package hk.collaction.timber.ui.activity;

import android.os.Bundle;
import android.support.v4.app.Fragment;
import android.support.v4.app.FragmentManager;
import android.support.v7.widget.Toolbar;

import butterknife.BindView;
import butterknife.ButterKnife;
import hk.collaction.timber.R;
import hk.collaction.timber.ui.fragment.SettingsFragment;

public class SettingsActivity extends BaseActivity {

	@BindView(R.id.toolbar)
	Toolbar toolbar;

	@Override
	public void onCreate(Bundle savedInstanceState) {
		super.onCreate(savedInstanceState);
		setContentView(R.layout.activity_container);

		ButterKnife.bind(this);
		setSupportActionBar(toolbar);

		initActionBar(getSupportActionBar(), R.string.title_activity_settings);

		Fragment mainFragment = SettingsFragment.newInstance();
		FragmentManager fragmentManager = getSupportFragmentManager();
		fragmentManager.beginTransaction()
				.replace(R.id.container, mainFragment)
				.commit();
	}

}

File: app/src/main/java/hk/collaction/timber/ui/settings/SettingsActivity.kt
package hk.collaction.timber.ui.settings

import androidx.fragment.app.Fragment
import hk.collaction.timber.R
import hk.collaction.timber.databinding.ActivityContainerBinding
import hk.collaction.timber.ui.base.BaseFragmentActivity

class SettingsActivity : BaseFragmentActivity<ActivityContainerBinding>() {
    override fun getActivityViewBinding() = ActivityContainerBinding.inflate(layoutInflater)
    override var fragment: Fragment? = SettingsFragment.newInstance()
    override var titleId: Int? = R.string.title_activity_settings
}